test/srt/openai_server/features/test_enable_thinking.py

```diff
@@ -189,7 +189,7 @@ def test_stream_chat_completion_without_reasoning(self):
         )
 
 
-## Skip for ci test
+# Skip for ci test
```

Comment:
Do we want to skip the test or is this a typo?

Reply:
Just skip the CI test. Model is too huge (450B). It is way too heavy to run in CI test. Just adding the test script here for others to quickly run the validation. It is similar to what we did for llama4: https://github.com/sgl-project/sglang/blob/484d0e021dcf825e692c1912365ed2a6cd3d947a/test/srt/test_vision_openai_server_b.py#L105